### ISSUE TITLE
Look for missing fire hydrants in Jura, France

### ISSUE THREAD

**Issue by pyrog** [osm-fr/osmose-backend#543]
Open Data for fire hydrants in Jura are available in several format and projections:
https://www.ideobfc.fr/geonetwork/srv/fre/catalog.search#/metadata/79884cd2-9ffb-44bb-b610-4572df87d0b2

catpei | osm tags
--|--
POTEAU INCENDIE|emergency=fire_hydrant<br/>fire_hydrant:type=pillar
BOUCHE INCENDIE|emergency=fire_hydrant<br/>fire_hydrant:type=underground
RESERVE ARTIFICIELLE|emergency=water_tank
RESERVE NATURELLE|emergency=water_pond

dispopei | osm tags
--|--
DISPONIBLE|emergency=*
DISPONIBLE RESTREINT|emergency=*
INDISPONIBLE|disused:emergency=*

typepei| osm tags
--|--
BI 100|fire_hydrant:diameter=100 // underground hydrant
NR|non renseigné ?
PA|// poteau d'aspiration
PA 100|fire_hydrant:diameter=100
PA 150|fire_hydrant:diameter=150
PI|// poteau incendie
PI 70|fire_hydrant:diameter=70
PI 80|fire_hydrant:diameter=80
PI 100|fire_hydrant:diameter=100
PI100|fire_hydrant:diameter=100
PI 150|fire_hydrant:diameter=150
RAA|// réserve artificielle aérienne ?<br/>// water tank (reservoir)
RAE|// réserve artificielle enterrée ?<br/>/ underground water tank
RAS|// réserve artificielle souterraine ?<br/>// floppy water tank
RN|// réserve naturelle  ?<br/>|// pond, lac, river, canal

numsdis| osm tags
--|--
v |ref:SDIS39=v

(An old? [csv parser for hydrants in switzerland](https://github.com/osm-fr/osmose-backend/blob/5c71d352837622a89d2e212872a2514b221bc604/analysers/analyser_merge_hydrant_point_ch.py) is available)

```python
        # CSV header :
        #gid,numcarto,numsdis,catpei,dispopei,typepei,insee,adressepei,precigeo,dernierct,derniervi,debit,pression,volume,diamcond,anomalie1,anomalie2,anomalie3,anomalie4,anomalie5,anomalie6,datadate

        self.defaultTag = {
            "source": "SDIS 39 - 2019"
        }
        self.defaultTagMapping = {
            "ref:SDIS39": "numsdis",
        }

```

**Comment by frodrigo** [comment]
Duplicate #413 
All available data source should be listed and if it's normed or not.

**Comment by frodrigo** [comment]
Can you try to write a full analyser, based on other codes? even if you do not test it.

**Comment by pyrog** [comment]
Yes.

It seem that Osmose import only files in CSV format ?
(OD in CSV don't have lat, lon, but fine in geojson, shp...)

Is it possible to test simply the analyzer ?

**Comment by frodrigo** [comment]
No, Osmose can load from geojson or shapefile: https://github.com/osm-fr/osmose-backend/blob/master/analysers/Analyser_Merge.py#L465-L499

You can test the analyser with docker: https://github.com/osm-fr/osmose-backend/tree/master/docker
